components/board/orangepi/data.go

```diff
@@ -3,11 +3,49 @@ package orangepi
 import "go.viam.com/rdk/components/board/genericlinux"
 
 const (
-	opzero2 = "OrangePi Zero2"
-	op3lts  = "OrangePi 3 LTS"
+	opzero2  = "OrangePi Zero2"
+	op3lts   = "OrangePi 3 LTS"
+	opzero2w = "OrangePi Zero 2W"
 )
 
 var boardInfoMappings = map[string]genericlinux.BoardInformation{
+	opzero2w: {
+		// OP zero 2w user manual: https://drive.google.com/drive/folders/1KIZMMDBlqf1rKmOEhGH7_7A-COAgYoGZ
+		// Gpio pins can be found on page 131.
+		PinDefinitions: []genericlinux.PinDefinition{
+			{Name: "3", DeviceName: "gpiochip0", LineNumber: 264, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "5", DeviceName: "gpiochip0", LineNumber: 263, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "7", DeviceName: "gpiochip0", LineNumber: 269, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "8", DeviceName: "gpiochip0", LineNumber: 224, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "10", DeviceName: "gpiochip0", LineNumber: 225, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "11", DeviceName: "gpiochip0", LineNumber: 226, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "12", DeviceName: "gpiochip0", LineNumber: 257, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "13", DeviceName: "gpiochip0", LineNumber: 227, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "15", DeviceName: "gpiochip0", LineNumber: 261, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "16", DeviceName: "gpiochip0", LineNumber: 270, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "18", DeviceName: "gpiochip0", LineNumber: 228, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "19", DeviceName: "gpiochip0", LineNumber: 231, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "21", DeviceName: "gpiochip0", LineNumber: 232, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "22", DeviceName: "gpiochip0", LineNumber: 262, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "23", DeviceName: "gpiochip0", LineNumber: 230, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "24", DeviceName: "gpiochip0", LineNumber: 229, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "26", DeviceName: "gpiochip0", LineNumber: 233, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "27", DeviceName: "gpiochip0", LineNumber: 266, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "28", DeviceName: "gpiochip0", LineNumber: 265, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "29", DeviceName: "gpiochip0", LineNumber: 256, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "31", DeviceName: "gpiochip0", LineNumber: 271, PwmChipSysfsDir: "", PwmID: -1},
+			// When we can switch between gpio and pwm, this would have line number 267.
+			{Name: "32", DeviceName: "gpiochip0", LineNumber: -1, PwmChipSysfsDir: "300a000.pwm", PwmID: 1},
```

Comment:
you can include the line number on these, just made [this change](https://github.com/viamrobotics/rdk/commit/0adc7b9cb98fcc687100eb92b75edcb7c095f114) yesterday that allows for gpio output on pwm pins.